doc/nrf/drivers/wifi/nrf700x/nrf700x.rst

```diff
@@ -71,6 +71,14 @@ The nRF7002 DK uses QSPI whereas the nRF7002 EK uses SPI.
 
 To connect the nRF7002 EK to the SoC, the ``nrf7002ek`` shield is required.
 
+Configuration
+*************
+
+The nRF700X Wi-Fi driver has the following configurable options:
+
+.. options-from-kconfig::
+   :show-type:
```

Comment:
add `:only-visible:` in case we add hidden options in the future.

Reply:
Using of only-visible directive results in display of only the config names. The corresponding help and type info is good to have.

Reply:
Are you sure? IIUC, then its about not showing `hidden` Kconfig options.

Reply:
As per our discussion, I have removed "only-visible" directive.